.travis.yml

```diff
@@ -18,11 +18,11 @@ before_script:
 
 script:
   - docker exec -i test-nff-go go mod download
-  - docker exec -i test-nff-go make
+  - docker exec -i test-nff-go make -j
   # Build standalone examples
-  - docker exec -i test-nff-go bash -c "cd examples && make gopacketParserExample && cd .."
-  - docker exec -i test-nff-go bash -c "cd examples && make nffPktgen && cd -"
-  - docker exec -i test-nff-go make -C examples/dpi
+  - docker exec -i test-nff-go bash -c "cd examples && make -j gopacketParserExample && cd .."
+  - docker exec -i test-nff-go bash -c "cd examples && make -j nffPktgen && cd -"
```

Comment:
here, is 'cd -' really wanted? or it s a typo and it should be 'cd ..'?

Reply:
`cd -` returns to a previous directory $OLDPWD which `cd` command sets every time it is called. I think neither `cd -` nor `cd ..` because it all happens in child bash process.